.stylelintrc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "stylelint-config-standard"
+}
+  
```

Comment:
- [ ] We need to use [this](https://github.com/microverseinc/linters-config/blob/master/react-redux/.stylelintrc.json) version instead.

Reply:
Thank you 😄